be/test/CMakeLists.txt

```diff
@@ -1,25 +1,4 @@
-# Copyright 2021-present StarRocks, Inc. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-#[[
-Currently, the unit tests for be are divided into two executable files: 
-starrocks_test_part1 and starrocks_test_part2. 
-When we try to perform all the tests with one executable file, 
-the binary size of the code exceeds the limit, which leads to linkage errors.
-]]
-
-set(EXEC_FILES_PART1
+set(EXEC_FILES
```

Comment:
License header should be kept

Reply:
this is just a revert of the original PR. can add the license header later in a separate PR.